src/stylus/components/_dialogs.styl

```diff
@@ -0,0 +1,64 @@
+.dialog
+  z-index 1001
+  transition: .3s ease-in-out
+
+  .card
+    elevation(24)
+
+    &__title
+      padding 24px 24px 0
+
+    &__text
+      padding 20px 24px 24px
+
+    &__row--actions
+      border initial
+      padding: 8px 0
+
+      .btn
+        margin-right: 8px
+
+  &--fullscreen
+    width 100%
+    height 100%
+    position fixed
+
+    .toolbar
+      height 56px           // TODO: height with two lines of text: 80dp
+      padding 0 !important
+
+      &__title
+        font-size 20px      // TODO: convert from 20sp
+        padding: 20px 0     // TODO: convert from 20dp
+
+      .btn
+        &:first-child      // TODO: full width 72dp
+          width: 24px !important
```

Comment:
We should try to avoid important when possible. These can be changed by reducing the min-width which is on all buttons. min-width: 24px width: 24px or even just doing max-width: 24px.